### PR TITLE
Clean and default update

### PR DESCRIPTION
**Pull request by aburrell**
Updating clean and default routines to comply with new standards for Instruments.  See https://github.com/pysat/pysat/pull/574.

### CODE REVIEW

**Comment by aburrell** [comment]
@jklenzing the problem with this branch is that the test class expects the instruments to have an attribute known as `supported_tags`.  There are a few issues with this:

1) This is currently not a required Instrument attribute;
2) The value is defined two ways in most of these routines (one way when setting up the `list_files` routine and then another way for the `download` and `list_remote_files` routines).

Should this be added as a required instrument attribute?  If so, how should it be defined?  If not, we need to update the tests to take it as a kwarg input or something.

**Comment by jklenzing** [comment]
I've updated this in the other branch to be a kwarg. I don't think supported_tags are standardized throughout the code at the moment. Here it is specifically used for the local routines.

**Comment by rstoneback** [comment]
supported tags was originally started to support the use of generalized NASA methods. It isn't currently a pysat standard. I think some of the required attributes came about after supported tags.

**Comment by aburrell** [comment]
Ok, so this will work after PR #20 is merged?

**Comment by jklenzing** [comment]
Just verified that #20 runs with the delimiter-bug branch of `pysat`.  The only errors are related to the `keep_original_names` keyword in the ICON instruments, which I think is fixed here.

**Comment by aburrell** [comment]
It looks like the change in coverage is caused by the different options for `default`.

File: pysatNASA/instruments/omni_hro.py [hunk 1]
```diff
@@ -57,45 +58,25 @@
 from pysat.instruments.methods import general as mm_gen
 from pysatNASA.instruments.methods import cdaweb as cdw
 
-import logging
 logger = logging.getLogger(__name__)
```
[the remark this answers is not in the document]
Comment (aburrell) [reply pysatNASA/instruments/omni_hro.py:61]: Yes, I think we should probably do that everywhere.

[the remark this answers is not in the document]
Comment (jklenzing) [reply pysatNASA/instruments/omni_hro.py]: fixing

**Review by jklenzing — APPROVED** [review]
Reduction in coverage consistent with new code styles (1 covered line deleted in each instrument).

**Pull request merged**